packages/light-client/vite.config.ts

```diff
@@ -0,0 +1,29 @@
+import {defineConfig, mergeConfig} from "vite";
+import {getBaseViteConfig} from "../../vite.base.config.js";
+
+import pkgJSON from "./package.json";
+
+export default mergeConfig(
+  getBaseViteConfig(pkgJSON, {libName: "LightClient", entry: "src/index.browser.ts"}),
+  defineConfig({
+    build: {
+      rollupOptions: {
+        output: {
+          footer: `
+        globalThis.lodestar = globalThis.lodestar === undefined ? {} : globalThis.lodestar;
+        globalThis.lodestar.lightclient = {
+          Lightclient,
+          LightclientEvent,
+          RunStatusCode,
+          upgradeLightClientFinalityUpdate,
+          upgradeLightClientOptimisticUpdate,
+          utils: index$1,
+          transport: index,
```

Comment:
why are these strange?

Reply:
Yes that's an ugly hack, could not find better workaround. The bundle have exports with the file names, as `utils/index.ts` and `transport/index.ts` have same file names, so it endup those weird looking export names.